packages/suite-data/files/translations/es.json

```diff
@@ -0,0 +1,1489 @@
+	"TR_GOT_IT": "¡Entendido!",
+	"TR_GO_TO_EXTERNAL_WALLET": "Ir a la wallet externa",
+	"TR_GO_TO_FIRMWARE": "Ir al firmware",
+	"TR_GO_TO_ONBOARDING": "Ir al tour de bienvenida",
+	"TR_GO_TO_SECURITYButton in security page (start security setup)": "Continuar con la copia de seguridad",
+	"TR_GO_TO_SETTINGS": "Ir a la configuración",
+	"TR_GO_TO_SUITE": "Ir a Suite",
+	"TR_GRAPH_MISSING_DATA": "Las transacciones Ripple y las transacciones de tokens en la red ethereum están excluidas del gráfico de transacciones",
+	"TR_GUIDE_ARTICLES": "Artículos",
+	"TR_GUIDE_BUG_LABEL": "Reportar un comportamiento defectuoso",
+	"TR_GUIDE_CATEGORIES": "Categorías",
+	"TR_GUIDE_DASHBOARD": "Dashboard",
+	"TR_GUIDE_FEEDACK_OR_SUGGESTION": "Comentarios y sugerencias",
+	"TR_GUIDE_FEEDBACK_BUG_TEXT_HEADLINE": "¿Cuál es el problema?",
+	"TR_GUIDE_FEEDBACK_CATEGORY_HEADLINE": "¿Cuál es la categoría?",
+	"TR_GUIDE_FEEDBACK_ERROR": "Ha ocurrido un error en el servidor. Inténtalo de nuevo más tarde.",
+	"TR_GUIDE_FEEDBACK_RATING_HEADLINE": "¿Disfrutando de Trezor Suite?",
+	"TR_GUIDE_FEEDBACK_SEND_REPORT": "Enviar informe",
+	"TR_GUIDE_FEEDBACK_SENT": "El mensaje ha sido enviado. ¡Gracias!",
+	"TR_GUIDE_FEEDBACK_SUGGESTION_TEXT_HEADLINE": "¿Cómo podemos mejorar?",
+	"TR_GUIDE_FEEDBACK_SYSTEM_INFO_NOTICE": "La información básica del sistema se compartirá anónimamente",
+	"TR_GUIDE_SUGGESTION_LABEL": "Solicitar una nueva funcionalidad",
+	"TR_GUIDE_VIEW_HEADLINE_HELP_US_IMPROVE": "Ayúdanos a mejorar",
+	"TR_GUIDE_VIEW_HEADLINE_LEARN_AND_DISCOVER": "Aprende y descubre",
+	"TR_GUIDE_VIEW_HEADLINE_REPORT_BUG": "Informa de un error",
+	"TR_GUIDE_VIEW_HEADLINE_SUGGEST": "Sugerencia",
+	"TR_HELP": "Ayuda",
+	"TR_HELP_TREZOR_SUITE": "Ayuda a Trezor Suite a mejorar",
+	"TR_HELP_TREZOR_SUITE_TEXT_1": "Ayuda a Trezor Suite a convertirse en un producto mejor enviándonos {TR_HELP_TREZOR_SUITE_TEXT_1_FAT}.",
+	"TR_HELP_TREZOR_SUITE_TEXT_1_FAT": "datos de análisis anónimos.",
+	"TR_HELP_TREZOR_SUITE_TEXT_2": "Trezor Suite NO rastrea ningún dato personal o relacionado con el balance.",
+	"TR_HIDDEN_WALLET": "Wallet oculta #{id}",
+	"TR_HIDDEN_WALLET_DESCRIPTION": "Requiere frase de contraseña",
+	"TR_HIDDEN_WALLET_MODAL_DESCRIPTION": "Extensión de la wallet derivada de la semilla de recuperación por tu propia entrada elegida.",
+	"TR_HIDDEN_WALLET_TOOLTIP": "Extension of the wallet derived from recovery seed by your own chosen input.",
+	"TR_HIDE_ADVANCED_OPTIONSHide advanced sending form": "Ocultar opciones avanzadas",
+	"TR_HIDE_BUTTON": "Ocultar",
+	"TR_HIDE_GRAPH": "Ocultar gráfico",
+	"TR_HOLD_BOTH_BUTTONS": "Hold both buttons while connecting device",
+	"TR_HOLD_LEFT_BUTTON": "Mantén pulsado el botón de la izquierda o ambos botones mientras conectas el dispositivo",
+	"TR_HOLOGRAM_STEP_ACTION_NOT_OKButton to click when hologram looks different": "Mi holograma parece diferente",
+	"TR_HOLOGRAM_STEP_ACTION_OKButton to click in alright case": "Mi holograma está bien",
```

Comment:
These translation keys doesn't look good. I guess that the transforming script could be broken and handled translations with description incorrectly.

Reply:
It's true, are good in Crowdin but not here.

Reply:
uf, thanks, good catch. It should be fixed now

Reply:
Looks good.